themes/default/layouts/page/pulumi-for-startups.html

```diff
@@ -0,0 +1,83 @@
+{{ define "hero" }}
+    <header class="page-hero">
+        <div class="dot-background-container">
+            <div class="dot-background"></div>
+        </div>
+        <div class="dot-overlay"></div>
+        <div class="page-hero-title">
+            <h1 class="rainbow-text mb-0">Pulumi for Startups</h1>
+        </div>
+    </header>
+{{ end }}
+
+{{ define "main" }}
+    <h2 class="mt-8 text-center">Get <b>$10,000</b> in Pulumi credits and exclusive help for your projects</h2>
+
+    <section id="success-stories" class="text-center p-8">
+        <div class="card mx-auto w-full lg:w-1/2 p-8">
+            <h3 class="text-gray-700 text-3xl font-medium">Pulumi Startup Success Stories</h3>
+            <div class="flex lg:flex-row flex-wrap justify-center space-x-4 spacy-y-4">
+                <a href="/case-studies/sourcegraph/"><img src="/logos/customers/sourcegraph-logo-cropped.svg" class="h-6 lg:h-8 my-2" alt="Sourcegraph" /></a>
+                <a href="/case-studies/panther-labs/"><img src="/logos/customers/panther-labs-wordmark.svg" class="h-6 lg:h-8 my-2" alt="Panther" /></a>
+                <img src="/logos/customers/cockroach-labs-logo.svg" class="h-6 lg:h-8 my-2" alt="CockroachDB" />
```

Comment:
this seems to be missing the link to the cockroach case study

Reply:
Yeah it's a little odd to have one missing link, but we don't currently have a cockroach case study to link to